diff --git a/node_modules/@codemirror/view/dist/index.cjs b/node_modules/@codemirror/view/dist/index.cjs
index 8b0473c..bb16f53 100644
--- a/node_modules/@codemirror/view/dist/index.cjs
+++ b/node_modules/@codemirror/view/dist/index.cjs
@@ -96,7 +96,7 @@ function scrollRectIntoView(dom, rect) {
     let doc = dom.ownerDocument, win = doc.defaultView;
     for (let cur = dom.parentNode; cur;) {
         if (cur.nodeType == 1) { // Element
-            let bounding, top = cur == document.body;
+            let bounding, top = cur == doc.body;
             if (top) {
                 bounding = windowRect(win);
             }
diff --git a/node_modules/@codemirror/view/dist/index.js b/node_modules/@codemirror/view/dist/index.js
index f134e96..bbeabd4 100644
--- a/node_modules/@codemirror/view/dist/index.js
+++ b/node_modules/@codemirror/view/dist/index.js
@@ -93,7 +93,7 @@ function scrollRectIntoView(dom, rect) {
     let doc = dom.ownerDocument, win = doc.defaultView;
     for (let cur = dom.parentNode; cur;) {
         if (cur.nodeType == 1) { // Element
-            let bounding, top = cur == document.body;
+            let bounding, top = cur == doc.body;
             if (top) {
                 bounding = windowRect(win);
             }
