diff --git a/node_modules/vfile-matter/index.js b/node_modules/vfile-matter/index.js
index 434bf0b..f91842f 100644
--- a/node_modules/vfile-matter/index.js
+++ b/node_modules/vfile-matter/index.js
@@ -10,7 +10,7 @@ function matter(file, options) {
   var strip = settings.strip
   var yamlOptions = settings.yaml || {}
   var doc = String(file)
-  var match = /^---(?:\r?\n|\r)(?:([\s\S]*)(?:\r?\n|\r))?---(?:\r?\n|\r|$)/.exec(
+  var match = /^---(?:\r?\n|\r)(?:([\s\S]*?)(?:\r?\n|\r))?---(?:\r?\n|\r|$)/.exec(
     doc
   )
 
